app/index.js

```diff
@@ -84,7 +88,7 @@ module.exports = yeoman.generators.Base.extend({
       if (err) return done(err);
       this.templates = list.map(function(t) {
         return {
-          name: util.format('%s (%s)', t.name, t.description),
+          name: g.format('%s (%s)', t.name, t.description),
```

Comment:
IMO, we need to ensure that at least `t.description` is localized. This probably means enhancing loopback-workspace with globalization support.